tests/test_code_list.py

```diff
@@ -118,3 +118,65 @@ def test_to_graph_should_return_subject() -> None:
     g2 = Graph().parse(data=src, format="turtle")
 
     assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_code_list_reference() -> None:
+    """It returns an identifier graph isomorphic to spec."""
+    codelist1 = CodeList()
+    codelist1.identifier = "http://example.com/codelists/1"
+
+    codelist2 = CodeList()
+    codelist2.identifier = "http://example.com/codelists/2"
+
+    codelist1.code_list_reference = codelist2
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
+        @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+        @prefix xkos: <http://rdf-vocabulary.ddialliance.org/xkos#> .
+
+
+        <http://example.com/codelists/1>
+                a modelldcatno:CodeList;
+                    modelldcatno:codeListReference <http://example.com/codelists/2> .
+
+        <http://example.com/codelists/2> a modelldcatno:CodeList .
```

Comment:
En slik referanse vil vel hovedsaklig bety eksterne kodelister, dette vil bare ende opp med en del tomme ressurser

Reply:
Nå må du være litt tydeligere på hva du vil ha endret.. ;-)

Reply:
Hvis jeg vil referere til den eksterne kodelisten `http://example.com/codelists/2` så har jeg ikke lyst at det opprettes en ressurs for den kodelisten her. Vil bare ha referansen.

Reply:
Dvs ikke opprette `<http://example.com/codelists/2> a modelldcatno:CodeList .` når det eneste som ligger på objektet er en identifier

Reply:
Dette hadde strengt tatt vært en bra ting for samtlige objekt

Reply:
Ok, jeg skjønner. Hva synes du @stigbd ? Jeg kan endre dette her, men det vil bli mer jobb å endre dette i samtlige objekt. Så det må jeg i så fall skrive opp på listen over ting som skal endres (teknisk gjeld). Og så er det spørsmålet om kanskje noen ting skal være med? De som faktisk er bnodes, kanskje? Men ikke de som har en identifikator?

Reply:
Samtlige trenger du ikke ta nå, men jeg ser ikke for meg at `modelldcatno:codeListReference` blir brukt til stort annet enn å referere til eksterne lister. Så her vil jeg si det er viktig

Reply:
Ok, da gjør jeg det. Endrer dette her nå og så setter jeg opp en task på å gjøre resten senere. Men jeg forstår det slik at når det finnes en identifikator på det eksterne objektet så skal den ikke med. Men hva hvis det faktisk er en BNode? Da skal den jo med, ikke sant?

Reply:
dvs et helt tomt objekt? Dunno, du kan vel like så greit la det bli oppretta som en BNode, der har en jo uansett valget med å ikke opprette objektet om en ikke vil ha det med.

Reply:
Ok, takk for tilbakemeldinger. Vi får ta den avklaringen når oppgaven skal utføres. Har satt opp sak her:

https://github.com/Informasjonsforvaltning/modelldcatnotordf/issues/63

Reply:
Eg har lagt inn kommentar på korleis eg meiner dette bør opprettes på det nye issuet her:
https://github.com/Informasjonsforvaltning/modelldcatnotordf/issues/63

Reply:
Hei dere, takk for tilbakemeldinger, jeg så ikke den siste kommentaren til @stigbd før nå. :blush: 

Men da skal jeg ta med meg dette når jeg går i gang med #63